loleaflet/src/control/Control.JSDialogBuilder.js

```diff
@@ -1887,14 +1887,18 @@ L.Control.JSDialogBuilder = L.Control.extend({
 			$(childContainer).hide();
 
 			if (builder.wizard) {
-				L.DomUtil.remove($(container).find('.loleaflet-annotation-menubar')[0]);
-				var arrowSpan = L.DomUtil.create('span','sub-menu-arrow',$(container).find('.loleaflet-annotation-content-wrapper')[0]);
-				arrowSpan.innerHTML = '>';
-				arrowSpan.style.padding = '0px';
-
+				if (data.children.length >= 2) {
+					L.DomUtil.remove($(container).find('.loleaflet-annotation-menubar')[0]);
+					var arrowSpan = L.DomUtil.create('span','sub-menu-arrow',$(container).find('.loleaflet-annotation-content-wrapper')[0]);
+					arrowSpan.innerHTML = '>';
+					arrowSpan.style.padding = '0px';
+
+					$(container).click(function() {
+						builder.wizard.goLevelDown(childContainer);
+					});
+				}
 				$(container).click(function() {
```

Comment:
Now, this block is common in both the conditional block. Why not to remove the else block and just have simple click event without any condition